src/front/styles/pattern-library.scss

```diff
@@ -389,34 +415,47 @@ hr.hr-strong {
     font-weight: 200;
     line-height: 16px;
 
+    > li {
+        font-size: 16px;
+    }
+
     > li > a {
         color: $pl-sidebar-link-fg;
         &:hover,
         &:focus {
-            background: $pl-sidebar-link-active-bg;
-            color: $pl-sidebar-link-fg;
+            background: $pl-sidebar-active-bg;
+            color: $white;
         }
         padding-left: 30px;
     }
 
     > li.active > a {
-        background: $pl-sidebar-link-active-bg;
-        color: $pl-sidebar-link-fg;
+        background: $white;
+        color: $pl-sidebar-bg;
+        border-right: 1px solid $pl-sidebar-bg;
     }
 
     > li.active > ul {
         background: $pl-sidebar-link-active-bg;
+        padding: 8px 0px;
     }
 
-    > li > ul > li > a {
-        padding-left: 40px;
+    > li > ul > li {
+        width: 98%;
+        margin: 0 auto;
     }
 
+    // > li > ul > li > a {
```

Comment:
Remove commented out code

Reply:
good catch, just removed and pushed updated file.